src/components/description/styles.module.css

```diff
@@ -0,0 +1,39 @@
+.descriptionContainer {
+    width: 70vw;
+    max-width: 300px;
+    margin: 1rem auto;
+}
+
+.descriptionHeading {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
```

Comment:
heading and icon should be center aligned

Reply:
done